admin-part/src/main/kotlin/Methods.kt

```diff
@@ -117,7 +117,7 @@ internal fun TypedRisks.toListDto(): List<RiskDto> = flatMap { (type, methods) -
         RiskDto(
             type = type,
             ownerClass = method.ownerClass,
-            name = method.name,
+            name = method.ownerClass.methodName(method.name),
```

Comment:
check can we make it in the first step to avoid invoke this method 4 times

Reply:
yeap